src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -309,6 +309,16 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
           return;
         }
 
+        if (event.name === 'applyFilter') {
```

Comment:
this should go under `if (!this.input.disableTriggers) `

Reply:
Agree!